ADAL/src/request/ADWebAuthResponse.m

```diff
@@ -25,14 +25,15 @@
 #import "ADWebAuthResponse.h"
 #import "ADWebResponse.h"
 #import "ADWebAuthRequest.h"
-#import "ADWorkplaceJoinConstants.h"
-#import "ADPKeyAuthHelper.h"
 #import "ADClientMetrics.h"
 #import "NSString+MSIDTelemetryExtensions.h"
 #import "MSIDTelemetryEventStrings.h"
+#import "MSIDPkeyAuthHelper.h"
 
 @implementation ADWebAuthResponse
 
+static NSString *const kPKeyAuthName = @"PKeyAuth";
```

Comment:
nit: don't we have a constant for this in common core?

Reply:
Added an issue to support PKeyAuth in silent, we can refactor then once complete.